### ISSUE TITLE
bower install for version 0.5.1 is not available. 

### ISSUE THREAD

**Issue by lmssrinivas** [s-yadav/angulargrid#33]
Can you update angulargrid @0.5.1 in releases

**Comment by mfilotto** [comment]
There is only version 0.4.0 in bower
````
{ ~ }  » bower info angulargrid                                                                 ~
bower angulargrid#*             cached https://github.com/s-yadav/angulargrid.git#0.4.0
bower angulargrid#*           validate 0.4.0 against https://github.com/s-yadav/angulargrid.git#*
````

**Comment by bboyle** [comment]
+1

bower should pick up the v0.5.1 release if it is tagged in github: https://github.com/s-yadav/angulargrid/releases

**Comment by s-yadav** [comment]
It required a fix, Added now 0.5.2 .
